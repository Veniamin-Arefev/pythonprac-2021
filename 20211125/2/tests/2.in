áûë òåêñò íà ðóññêîì â êîäèðîâêåsdasdasdasdsdasdasdфывфывфыв