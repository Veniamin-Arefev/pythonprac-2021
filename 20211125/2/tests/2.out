был текст на русском в кодировкеsdasdasdasdsdasdasd?????????
